hydra/core/utils.py

```diff
@@ -151,7 +150,7 @@ def register(name: str, f: Any) -> None:
         lambda path: OmegaConf.select(cast(DictConfig, HydraConfig.get()), path),
     )
     register(
-        "python_version", lambda: platform.python_version(),
+        "python_minor_version", lambda: f"{sys.version_info.major}.{sys.version_info.minor}",
```

Comment:
odd.

Reply:
yes, i ran into some issues with CircleCI. This was an old version of the code that I thoguth would fix.

I'm going to do some further clean up :)